apps/app/components/modules/single-module-card.tsx

```diff
@@ -40,14 +43,33 @@ export const SingleModuleCard: React.FC<Props> = ({ module }) => {
         data={module}
       />
       <div className="group/card h-full w-full relative select-none p-2">
-        <div className="absolute top-4 right-4 z-10 bg-red-200 opacity-0 group-hover/card:opacity-100">
-          <button
-            type="button"
-            className="grid h-7 w-7 place-items-center  bg-white p-1 text-red-500 outline-none duration-300 hover:bg-red-50"
-            onClick={() => handleDeleteModule()}
-          >
-            <TrashIcon className="h-4 w-4" />
-          </button>
+        <div className="absolute top-4 right-4 ">
+          <CustomMenu width="auto" ellipsis>
+            <CustomMenu.MenuItem
+              onClick={() =>
+                copyTextToClipboard(
+                  `https://app.plane.so/${workspaceSlug}/projects/${projectId}/modules/${module.id}`
```

Comment:
using an absolute path like this will create an issue. in stage, it will copy the production link.

Reply:
Okay, I get it. I've fixed it. Now, instead of using an absolute path, I'm using the origin.